FusionIIIT/applications/academic_procedures/views.py

```diff
@@ -679,11 +683,12 @@ def branch_change_request(request):
                 extraInfo_user - gets the user details from the extrainfo model.
                 department - user's branch.
```

Comment:
This is not the user's branch, instead, the branch, user applied to.

Reply:
Updated